### ISSUE TITLE
Text fields prefix show error

### ISSUE THREAD

**Issue by nuintun** [vuetifyjs/vuetify#1620]
### Versions
0.15.2

When text fields set ```prefix``` option:

![qq 20170907112043](https://user-images.githubusercontent.com/1730277/30144716-87cf853c-93bf-11e7-9e4e-3b8810bad6bd.png)
![qq 20170907112053](https://user-images.githubusercontent.com/1730277/30144719-8c8b793c-93bf-11e7-98c9-d7c097ef80c1.png)

### Reproduction Link
[https://codepen.io/anon/pen/QMRBBN](https://codepen.io/anon/pen/QMRBBN)

**Comment by johnleider** [comment]
Due to how the label is constructed, I'm not sure if this can be supported. You may have to forgot the label and use placeholder instead.

**Comment by roryprimrose** [comment]
Just hit this one too and was about to raise an issue. Codepen at https://codepen.io/anon/pen/boVWYX

**Comment by peluprvi** [comment]
And the label's left space got wrong too if you add a placeholder: https://codepen.io/anon/pen/gGrGwo

**Comment by nuintun** [comment]
I think can hidden the prefix and suffix when the text fields is empty ~

**Comment by johnleider** [comment]
@nuintun That might have to be the solution to this for the short term.

**Comment by Maxhodges** [comment]
another reproduction link
text-field label should make room for :prefix content
https://codepen.io/maxhodges/pen/EojwPV?editors=1010

Select a country. CountryCode is prepended to the phone text field, but it collides with the label. I think the label should move out of the way (up or over)  for prepended text.

**Comment by Maxhodges** [comment]
oddly the label DOES move out the way in the docs
![image](https://user-images.githubusercontent.com/957271/34072806-582ecb42-e2d1-11e7-86ef-900696d913a9.png)

**Comment by Maxhodges** [comment]
here's how I've dealing with it for now

Use `placeholder` for a label, then on text entry `placeholder` is replaced with a `label`
```
  <v-text-field prepend-icon="phone" placeholder="Phone Number" :label="phoneNumber ? 'Phone Number' : ''" :prefix=" country ? country.countryCode : '' " v-model="phoneNumber "></v-text-field>
```

**Comment by johnleider** [comment]
This issue has been resolved as of 0.17 branch.

**Comment by johnleider** [comment]
We kindly ask users to not comment on closed/resolved issues. If you believe that this issue has not been correctly resolved, please create a new [issue](https://issues.vuetifyjs.com) showing the regression or reach out to us in our [Discord community](https://community.vuetifyjs.com).

Thank you for your contribution and interest in improving Vuetify.
